src/webapi/events/mouse.rs

```diff
@@ -195,6 +196,8 @@ pub enum MouseButton {
     Button4,
     /// The fifth mouse button (browser forward).
     Button5,
+    /// The sixth mouse button, or the Pen Eraser button
+    Eraser,
```

Comment:
Adding a new variant is also technically a breaking change, so for now could you comment this out and leave a `TODO`?